### PR TITLE
Add background music to the page.

### PR DESCRIPTION
**Pull request by koenbollen**
For a full immersive experience we need some background music, Joost's favourite music has been selected.
- [x] Fix looping
- [x] Fix `&` encoding, replace with `&amp;`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,5 +12,6 @@
 
     <h1>NEE</h1>
     
+    <iframe width="0" height="0" src="http://www.youtube.com/embed/nQyLxlZDMZY?rel=0&autoplay=1" frameborder="0" allowfullscreen></iframe>
```
Comment (jurriaan) [inline index.html]: `&amp;` ?

Comment (jurriaan) [inline index.html]: https://www.youtube.com/v/nQyLxlZDMZY?autoplay=1&loop=1&playlist=nQyLxlZDMZY incl loop

Comment (rickpastoor) [inline index.html]: This looks good to me. `&` will be correctly served in an UTF-8 document I presume?

Comment (jurriaan) [inline index.html]: http://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp

**Comment by koenrh** [comment]
:+1:

**Comment by rickpastoor** [comment]
I'm so happy with this. We debated this for many seconds, great to see it finally added to the project!

File: index.html (hunk 1 above)
Comment (rickpastoor) [inline index.html]: Ah you're right.

**Comment by jurriaan** [comment]
To quote @rickpastoor: "Thinking big is essential to extraordinary results."

**Comment by koenbollen** [comment]
I'm currently working on looping the music.

**Comment by jurriaan** [comment]
Awesome! :dancer:

**Comment by rickpastoor** [comment]
I added a checklist to keep track of the requirements.

**Comment by rickpastoor** [comment]
Nice quote!

**Comment by jurriaan** [comment]
LGTM!

**Pull request merged**